Warning, the following versions were found for oauthlib
0.7.2
0.7.1
0.7.0
0.6.3
0.6.2
0.6.1
0.6.0
0.5.1
0.5.0
0.4.2
0.4.1
0.4.0
0.3.8
0.3.7
0.3.6
0.3.5
0.3.4
0.3.3
0.3.2
0.3.0
0.1.3
0.1.2
0.1.1
0.1.0
0.0.2
0.0.1
Using 0.7.2
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/o/oauthlib/oauthlib-0.7.2.tar.gz (104 KB) for oauthlib.
Downloading oauthlib (use --no-download to skip this step)
Using cached download
Unpacking oauthlib...
done
working in /tmp/tmpjm928Aconda_skeleton_oauthlib-0.7.2.tar.gz
Fetching package metadata: ......
Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).

The following NEW packages will be INSTALLED:

    numpy:      1.9.2-py27_0
    openssl:    1.0.1k-1    
    pip:        7.0.3-py27_0
    python:     2.7.10-0    
    pyyaml:     3.11-py27_0 
    readline:   6.2-2       
    setuptools: 17.0-py27_0 
    sqlite:     3.8.4.1-1   
    system:     5.8-2       
    tk:         8.5.18-0    
    yaml:       0.1.4-0     
    zlib:       1.2.8-0     

Linking packages ...
[                    ]|                                                  |   0%[openssl             ]|                                                  |   0%[readline            ]|####                                              |   8%[sqlite              ]|########                                          |  16%[system              ]|############                                      |  25%[tk                  ]|################                                  |  33%[yaml                ]|####################                              |  41%[zlib                ]|#########################                         |  50%[numpy               ]|#############################                     |  58%[python              ]|#################################                 |  66%[pyyaml              ]|#####################################             |  75%[setuptools          ]|#########################################         |  83%[pip                 ]|#############################################     |  91%[      COMPLETE      ]|##################################################| 100%
Applying patch: '/tmp/tmpjm928Aconda_skeleton_oauthlib-0.7.2.tar.gz/pypi-distutils.patch'
pyjwt not found, trying PyJWT
2
Warning, the following versions were found for PyJWT
1.3.0
1.1.0
1.0.1
1.0.0
0.4.3
0.4.2
0.4.1
0.4.0
0.3.2
0.3.1
0.3.0
0.2.3
0.2.1
0.2.0
0.1.9
0.1.8
0.1.7
0.1.6
0.1.5
0.1.4
0.1.3
0.1.2
0.1.1
Using 1.3.0
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/P/PyJWT/PyJWT-1.3.0.tar.gz (60 KB) for PyJWT.
Downloading PyJWT (use --no-download to skip this step)
Using cached download
Unpacking PyJWT...
done
working in /tmp/tmpj2aUBlconda_skeleton_PyJWT-1.3.0.tar.gz
Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).

Applying patch: '/tmp/tmpj2aUBlconda_skeleton_PyJWT-1.3.0.tar.gz/pypi-distutils.patch'
Warning, the following versions were found for pep8-naming
0.2.2
0.2.1
0.2
Using 0.2.2
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/p/pep8-naming/pep8-naming-0.2.2.tar.gz (5 KB) for pep8-naming.
Downloading pep8-naming (use --no-download to skip this step)
Using cached download
Unpacking pep8-naming...
done
working in /tmp/tmpd96CK2conda_skeleton_pep8-naming-0.2.2.tar.gz
Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).

Applying patch: '/tmp/tmpd96CK2conda_skeleton_pep8-naming-0.2.2.tar.gz/pypi-distutils.patch'
Warning, the following versions were found for flake8-import-order
0.6
0.5.3
0.5.2
0.5.1
0.5
0.4.3
0.4.2
0.4.1
0.4
0.3.2
0.3.1
0.3
0.2
0.1
Using 0.6
Use --version to specify a different version.
Using url https://pypi.python.org/packages/source/f/flake8-import-order/flake8-import-order-0.6.linux-x86_64.tar.gz (14 KB) for flake8-import-order.
Downloading flake8-import-order (use --no-download to skip this step)
Using cached download
Unpacking flake8-import-order...
done
working in /tmp/tmpZVZBfRconda_skeleton_flake8-import-order-0.6.linux-x86_64.tar.gz
Solving package specifications: .patching file core.py
Hunk #1 succeeded at 168 with fuzz 2 (offset 2 lines).
/home/ubuntu/miniconda/envs/_build/bin/python: can't open file 'setup.py': [Errno 2] No such file or directory

Applying patch: '/tmp/tmpZVZBfRconda_skeleton_flake8-import-order-0.6.linux-x86_64.tar.gz/pypi-distutils.patch'
$PYTHONPATH = /tmp/tmpZVZBfRconda_skeleton_flake8-import-order-0.6.linux-x86_64.tar.gz/home
Error: command failed: /home/ubuntu/miniconda/envs/_build/bin/python setup.py install
